src/lib/components/SubscribeForm/_styles.scss

```diff
@@ -0,0 +1,79 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/tools/breakpoints';
+
+web-subscribe-form {
+  display: block;
+  margin-top: 32px;
+  text-align: center;
+  width: 100%;
+
+  button {
```

Comment:
I think these will need BEM selectors like .web-subscribe-form__button. We try to avoid styling generic elements with descendant selectors because it can lead to specificity battles and accidental cascades.